### DISCUSSION TITLE
The "path" argument must be of type string or an instance of Buffer or URL. Received undefined

### DISCUSSION THREAD

**Discussion by AndonMitev** [shuding/nextra#2509] [General]
Hello, I'm having some issues during build process using:

```
"next": "14.0.0",
"nextra": "2.13.2",
"nextra-theme-docs": "2.13.2",
```

When i start building i get this error:

```
Failed to compile.

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received undefined
    at readdir (node:fs:1418:10)
```

`next.config.js`:

```
const withNextra = require('nextra')({
  theme: 'nextra-theme-docs',
  themeConfig: './theme.config.jsx'
})

module.exports = withNextra(nextConfig)
```

`theme.config.jsx` on the root level of the project aka the same as `next.config.js`

```
export default {
  logo: <span>My Nextra Documentation</span>,
  project: {
    link: 'https://github.com/shuding/nextra',
  },
  // ... other theme options
};

```

Any idea what might cause the issue?

**Comment by ariroseonline** [comment]
Nextra doesn't yet support the App router, but you can make a Pages directory side by side.

**Comment by caffeine-addictt** [comment]
@AndonMitev 

To add onto @ariroseonline's comment, you might find this https://github.com/shuding/nextra/issues/2688#issuecomment-1972156650 helpful.
